google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanelTest.java

```diff
@@ -47,23 +47,32 @@
 
   @Mock private Blob directoryBlob;
   @Mock private Blob binaryBlob;
+  @Mock private Blob nestedBlob;
 
-  private BiMap<Integer, String> indexToColName =
+  private static final String DIR_NAME = "my_directory";
+  private static final String BLOB_NAME = "my_blob.zip";
+  private static final String BLOB_CONTENT_TYPE = "application/zip";
+  private static final String NESTED_BLOB_FULL_NAME = "dir1/dir2/nested_blob.zip";
+  private static final String NESTED_BLOB_NAME = "nested_blob.zip";
+
+  private static final BiMap<Integer, String> indexToColName =
```

Comment:
nit: Use CONSTANT_CASE for these, now that they're constants.

Reply:
done